### DISCUSSION TITLE
Transparent background for videos

### DISCUSSION THREAD

**Discussion by yogesh9245** [plemeri/transparent-background#58] [General]
How can a transparent background be achieved in videos just like images ?

**Comment by yogesh9245** [comment]
I want to achieve transparent background in videos

**Reply by plemeri** [reply]
Do you specifically know any video format that supports RGBA encoding? To the best of my knowledge, there's no such thing. 

If you want to change the background into certain color, try using `--type '[0, 0, 0]'` which is a RGB format to change the background. `[0, 0, 0]` means black as you know, so change the value as you want. If you're saying a truly transparent in videos, then please convert video into images, then make them transparent frame by frame. 

It is not because our method cannot generate transparent background, it is because there are no such thing as RGBA video format which can represent transparency. Please understand this circumstances.

**Reply by yogesh9245** [reply]
okay got it.
I think going frame by frame would work

**Reply by plemeri** [reply]
Great! Thank you for your understanding. Please feel free to leave any question.
